### ISSUE TITLE
Unstable attributes sorting

### ISSUE THREAD

**Issue by gebsh** [prettier/plugin-xml#732]
The `xmlSortAttributesByKey` option is unstable when comparing an `xmlns` attribute with another prefixed attribute. For example, consider the following XML element:

```xml
<foo xmlns:foo="http://www.example.org" foo:foo="1" />
```

Formatting it with `prettier --plugin=@prettier/plugin-xml --write --xml-sort-attributes-by-key` swaps attributes:

```xml
<foo foo:foo="1" xmlns:foo="http://www.example.org" />
```

However, running the same command again, swaps the attributes once more, changing the code back to its original form.

The possible culprit of this behavior may be the following line:

https://github.com/prettier/plugin-xml/blob/b0a1dd7c1b99c9f70320f85f97044dd771db3476/src/printer.js#L431

There's a typo in the `if` statement—instead of `xmlns`, it's `xlmns`.

**Comment by kddnewton** [comment]
lol, good catch

**Comment by kddnewton** [comment]
Released with v3.2.2
